### ISSUE TITLE
Constantly crashing while listening to tracks or when adding to playlist

### ISSUE THREAD

**Issue by mariyanh** [nulloy/nulloy#225]
The player is constantly freezing and not responding while playing added tracks or when being added to the playlist.

2020 Macbook Pro 13 inch
8 GB RAM
M1 processor / Sonoma OS 14.2.1

Here are the custom settings that i've done, if there's a chance that it might be causing it:
 
![Screenshot 2024-02-22 at 10 41 01](https://github.com/nulloy/nulloy/assets/26903018/58d29c50-99a2-46bc-bff3-24567b75c036)

**Comment by mariyanh** [comment]
Thanks,

I just saw it after I posted. Installed and testing :)

Cheers!

**Comment by noscript** [comment]
Please report soon, I plan to make an official release out of it.

**Comment by noscript** [comment]
Can you please describe "freezing" in more details? Does the playback stop? Does player still react to mouse events (buttons change color when hovered)? Do you get a "application not responding" kind of dialog from the OS?

**Comment by mariyanh** [comment]
Exactly, music stops, player is not responding, mouse turns into the wheel of death :)
I force quit it, but no send a report is showing.

**Comment by noscript** [comment]
Please paste the report here, remove any sensitive information, if any.

**Comment by mariyanh** [comment]
sorry, but where do i find it ?

**Comment by noscript** [comment]
Sorry, I misread your message. I thought it suggested to send a report.

Was there any similar problem with 0.9.5 version? In 0.9.7 I rewrote the playback, which introduced many bugs unfortunately.

**Comment by mariyanh** [comment]
yes, I can see we've commented on the webiste release post with another guy as well..

**Comment by mariyanh** [comment]
Just to update here.
Unfortunately, It's still freezing/crashing.
System: M4, 32gb ram, OS: Tahoe 26.2 

Can't really notice if there is a specific pattern when it does.
